mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulPlayScreen.kt

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.sdp.mobile.state
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import ch.epfl.sdp.mobile.ui.play.PlayScreen
+import ch.epfl.sdp.mobile.ui.play.PlayScreenState
+
+/**
+ * Player class that holds info about chess player
+ * @param newGame Callable action for newGame
```

Comment:
I don't understand what the action do, can you add more detail in the doc ?

Reply:
Ofc